kornia/augmentation/augmentation.py

```diff
@@ -89,8 +89,11 @@ class RandomHorizontalFlip(GeometricAugmentationBase2D):
     def __repr__(self) -> str:
         return self.__class__.__name__ + f"({super().__repr__()})"
 
+    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor]:
+        return dict(batch_shape=torch.as_tensor(batch_shape))
```

Comment:
Can you test this under CUDA env? Our CUDA CI is currently down.

Reply:
It passed. My local pytorch version is (copy from `conda list`): 

pytorch                   1.9.0           py3.7_cuda10.2_cudnn7.6.5_0    pytorch